minimesos/src/main/java/com/containersol/minimesos/main/CommandUninstall.java

```diff
@@ -0,0 +1,72 @@
+package com.containersol.minimesos.main;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.containersol.minimesos.MinimesosException;
+import com.containersol.minimesos.cluster.ClusterRepository;
+import com.containersol.minimesos.cluster.Marathon;
+import com.containersol.minimesos.cluster.MesosCluster;
+import com.containersol.minimesos.mesos.MesosClusterContainersFactory;
+import java.io.PrintStream;
+
+/**
+ * Uninstalls a Marathon app or framework
+ */
+@Parameters(separators = "=", commandDescription = "Uninstall a Marathon app")
+public class CommandUninstall implements Command {
+
+    @Parameter(names = "--app", description = "Marathon app to uninstall", required = true)
+    private String app = null;
+
+    private ClusterRepository repository = new ClusterRepository();
+
+    private PrintStream output = System.out; // NOSONAR
+
+    public CommandUninstall(PrintStream output) {
+        this.output = output;
+    }
+
+    public CommandUninstall() {
+        // NOSONAR
+    }
+
+    @Override
+    public boolean validateParameters() {
+        return true;
+    }
+
+    @Override
+    public String getName() {
+        return "uninstall";
+    }
+
+    @Override
+    public void execute() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the declaration of thrown exception 'com.containersol.minimesos.MinimesosException' which is a runtime exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)